checks/security_policy.go

```diff
@@ -34,6 +34,9 @@ func SecurityPolicy(c *checker.CheckRequest) checker.CheckResult {
 		if strings.EqualFold(name, "security.md") {
 			logf("security policy : %s", name)
 			return true, nil
+		} else if strings.EqualFold(name, "**/security.rst") {
```

Comment:
@azeemshaikh38  The file glob pattern doesn't seem to work with the refactor. `security.rst` could be `docs` or other folders and it is not consistent across repositories.

Reply:
we are all out this week, can discuss in next week sync.

Reply:
OK